Assets/Scripts/Attributes/AAnimal.cs

```diff
@@ -0,0 +1,120 @@
+﻿using UnityEngine;
+using Ecosystem.StateMachines;
+using Ecosystem.ECS.Hybrid;
+
+namespace Ecosystem.Attributes {
+    public class AAnimal : MonoBehaviour, IAnimal {
+
+        private float Hunger;
+        //private float Thirst;
+        //private float Mating
+        private float HungerLimit;
+        //private float ThirstLimit;
+        //private float MatingLimit = Random.Range(0.3f,0.8f);
+
+        //public float Speed { get; set; }
+        //public float SprintSpeed { get; set; }
+        public Movement movement;
+        public Sensors Sensors;
```

Comment:
Not super important, but to follow the standard. This is almost equivalent.
```suggestion
        [SerializeField]
        private Movement movement;
        [SerializeField]
        private Sensors Sensors;
```